packages/toChars/src/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @description Splits input text into an array of characters.
+ * @param {String | Number} input
+ * @returns {Array}
+ * @example
+ * toChars('Hello world'); // => [ "H", "e", "l", "l", "o", " ", "w", "o", "r", "l", "d" ]
+ * toChars(2e5) // =>  ["2", "0", "0", "0", "0", "0"]
+ */
+
+const toChars = (input = []) => {
+  switch (input) {
+    case null:
+      return [];
+    case input:
```

Comment:
That's weird I would suggest removing the `switch/case` and maybe try something like:
```
const toChars = input => {
  if (!input) {
    return [];
  }
  return input.toString().split('');
}
```